packages/utils/src/io.ts

```diff
@@ -177,8 +177,7 @@ export async function isDirectory(path: string): Promise<boolean> {
   return (await stat(path)).isDirectory();
 }
 
-export const npmInstallFlags =
-  "--ignore-scripts --no-shrinkwrap --no-package-lock --no-bin-links --no-save --no-audit --no-fund --legacy-peer-deps";
+export const pnpmInstallFlags = "--ignore-scripts --no-save";
```

Comment:
Unless we're using pnpm to install other stuff, we don't need to set ignore-scripts as that's already in npmrc. But, not harmful.

Reply:
Yeah, in `typescript-installer.ts`, we're just using npm. It's possible that we just aren't doing enough installing anymore to warrant the constant?